e2e/cypress/integration/dcat_us_export.cy.js

```diff
@@ -44,9 +47,14 @@ describe('DCAT-US Export', () => {
             });
         });
         cy.exec('rm cypress/downloads/draft*', { failOnNonZeroExit: false });
+        cy.exec('rm cypress/downloads/redacted.zip', { failOnNonZeroExit: false });
+        cy.exec('rm cypress/downloads/data.json', { failOnNonZeroExit: false });
+
+        cy.login();
     });
 
-    afterEach(() => {
+    after(() => {
+        cy.logout();
```

Comment:
do you want logout `afterEach`?

Reply:
We should, but since we have cy.login() in beforeEach, and it does a cy.logout() implicitly, so no need to add the extra code. 
If we ever remove cy.login() from beforeEach, i would come back and add cy.logout() into afterEach.

Reply:
>  it does a cy.logout() implicitly

aha. yeah that makes sense.